plugins/generator-1.15.2/forge-1.15.2/procedures/world_use_bonemeal.java.ftl

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
These outer brackets are not really needed as if already wraps all statements into a whole unit.